src/tests/ftest/container/multiple_container_delete.py

```diff
@@ -30,22 +30,21 @@ def test_multiple_container_delete(self):
             times.
             Verify both the SCM and SSD pool spaces are recovered
 
-        :avocado: tags=all,hw,large,full_regression,container
-        :avocado: tags=multicontainerdelete
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,large
+        :avocado: tags=container,multi_container_delete
         """
-
-        if self.pool is None:
-            self.create_pool()
-        self.pool.connect()
+        self.add_pool(connect=False)
 
         out = []
 
         initial_scm_fs, initial_ssd_fs = self.get_pool_space()
 
-        for i in range(1000):
+        for i in range(500):
```

Comment:
Minor - Use case above says:
`Delete the container and repeat the above steps 1000`
Should now say
`Delete the container and repeat the above steps 500`

Reply:
I think 500 is overkill too. This is one of the longest tests in weekly, so hopefully this will at least make it faster. Thanks.

Reply:
I created https://daosio.atlassian.net/browse/DAOS-8623 to re-evaluate the requirements of this test.

Reply:
Thanks!

Reply:
@shimizukko Per https://daosio.atlassian.net/browse/DAOS-8623, it seems we could drastically reduce the number of iterations further. If you want, we could do that on this PR. I think as little as 50 iterations should be okay, with a test timeout of 600 (10 minutes)

Reply:
Done. Thank you for figuring out the improvements!